movies/src/main/java/comp.fundacionjala/movies/Movie.java

```diff
@@ -0,0 +1,33 @@
+package comp.fundacionjala.movies;
+
+class Movie {
+
+    private String title;
+    private int priceCode;
```

Comment:
Is it necessary to use the priceCode ?
because now you are using inheritance and you can take advantage of that